core-tests/shared/src/test/scala/zio/RefSpec.scala

```diff
@@ -45,29 +45,103 @@ object RefSpec
         testM("modify") {
           for {
             ref   <- Ref.make(current)
-            r     <- ref.modify[String](_ => ("hello", update))
+            r     <- ref.modify[Nothing, String](_ => ("hello", update))
             value <- ref.get
           } yield assert(r, equalTo("hello")) && assert(value, equalTo(update))
         },
         testM("modifySome") {
           for {
             ref   <- Ref.make[State](Active)
-            value <- ref.modifySome[String]("State doesn't change") { case Closed => ("active", Active) }
+            value <- ref.modifySome[Nothing, String]("State doesn't change") { case Closed => ("active", Active) }
```

Comment:
😱 
is there a way to get this inferred @regiskuckaertz  ?

Reply:
Not sure 🤔